packages/3id-did-resolver/src/index.ts

```diff
@@ -78,6 +79,22 @@ export function wrapDocument(content: any, did: string): DIDDocument | null {
   }, startDoc)
 }
 
+/**
+ * Return last anchor log entry, or genesis if no anchors found
+ */
+function lastAnchorOrGenesisEntry(log: LogEntry[]): LogEntry {
```

Comment:
Is recursion better than a for loop here?

Reply:
It is more idiomatic.

Reply:
changed to `for` loop anyway